README.md

```diff
@@ -123,6 +123,13 @@ GDS::SSO.config do |config|
 end
 ```
 
+### Use in production mode
+
+To use gds-sso in production you will need to setup the following environment variables, which we look for in [the config](lib/gds-sso/config.rb). You will need to have admin access to Signon to get these.
```

Comment:
Won't this parse to https://github.com/alphagov/gds-sso/lib/gds-sso/config.rb (which 404s)?

```suggestion
To use gds-sso in production you will need to setup the following environment variables, which we look for in [the config](https://github.com/alphagov/gds-sso/blob/master/lib/gds-sso/config.rb). You will need to have admin access to Signon to get these.
```

Reply:
It works for me - not sure what you mean by "parse".

Could you explain what's not working for you?

(I usually try to be specific about the commit/branch I'm referring to something on.)

Reply:
I think `[the config](lib/gds-sso/config.rb)` will parse as `<a href="https://github.com/alphagov/gds-sso/lib/gds-sso/config.rb">the config</a>`, which is a 404ing link.

Reply:
Oops - I was reading your suggestion as if it were what I'd written 😆.

Thanks for persisting - fixed.